admin_user.go

```diff
@@ -64,6 +64,14 @@ type AdminUserList struct {
 	Items []*AdminUser
 }
 
+// A list of relations to include. See available resources
+// https://www.terraform.io/docs/cloud/api/admin/users.html#available-related-resources
+type AdminUserIncludeOps string
+
+var (
```

Comment:
Yes, I can make that change. As you may have noticed , I have intentionally had some typed string declared as `var` and some as `const`. This is left-over from my previous approach when I was making the include field a typed string rather than a slice of typed strings. Because a `constant` is not "addressable" (&) for those instance where include was `*string`. Now it's no longer needed, yay.